clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -359,6 +358,25 @@ public String toString() {
         }
     }
 
+    public static final class ErrorRecord {
+        private final int relativeOffset;
```

Comment:
This is an internal class, so perhaps we can make these fields public and get rid of the getters.

Reply:
ack